src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/services/staff/DefendantResponseStaffNotificationServiceTest.java

```diff
@@ -82,9 +82,66 @@ public void shouldSendEmailWithExpectedContent() {
         );
     }
 
+    @Test
+    public void shouldSendEmailWithExpectedContentFullAdmissionPayImmediately() {
+        Claim claimWithFullAdmission = SampleClaim.builder()
+            .withResponse(SampleResponse.FullAdmission.builder().buildWithPaymentOptionImmediately())
+            .withRespondedAt(LocalDateTime.now())
+            .build();
+
+        service.notifyStaffDefenceSubmittedFor(claimWithFullAdmission, DEFENDANT_EMAIL);
+
+        verify(emailService).sendEmail(senderArgument.capture(), emailDataArgument.capture());
+
+        assertThat(emailDataArgument.getValue()
+            .getSubject()).startsWith("Pay immediately " + claimWithFullAdmission.getReferenceNumber());
```

Comment:
Is that what business wanted? Subject seems very generic to me especially that we have for example CCJ with immediate payments. I would expect something like `Full admission with immediate payment` or similar.

Reply:
This is what we have in story requirements.

Reply:
OK, happy with that. We might change that later when we have part admission as well but OK for now.